pom.xml

```diff
@@ -86,7 +88,7 @@
     <junit.vintage.version>5.6.1</junit.vintage.version>
     <mockito.jupiter.version>3.3.3</mockito.jupiter.version>
     <log4j.version>1.2.17</log4j.version>
-    <slf4j.version>1.7.5</slf4j.version>
+    <slf4j.version>1.7.15</slf4j.version>
```

Comment:
This is misleading. We have this still in the parent pom? Huh

Reply:
yes, filed a ticket